emcc.py

```diff
@@ -2276,10 +2283,6 @@ def phase_linker_setup(options, state, newargs):
 
   settings.REQUIRED_EXPORTS += ['stackSave', 'stackRestore', 'stackAlloc']
 
-  if not settings.STANDALONE_WASM:
-    # in standalone mode, crt1 will call the constructors from inside the wasm
-    settings.REQUIRED_EXPORTS.append('__wasm_call_ctors')
```

Comment:
Is this no longer needed for some reason?

Reply:
This was duplicated for some reason.     Just cleaning up.